### PR TITLE
Couple bugfixes

### PR DESCRIPTION
**Pull request by cadyn**
Adds NIF Responsive Filter protection from Portable Flashers, and fixes problems with food reagents being transferred into pred's bloodstream after prey is absorbed, causing toxic damage.

### CODE REVIEW

**Comment by TheFurryFeline** [comment]
I thought the reagents thing was intentional.

File: code/game/machinery/flasher.dm [hunk 1]
```diff
@@ -68,6 +68,9 @@
 		var/flash_time = strength
 		if(istype(O, /mob/living/carbon/human))
 			var/mob/living/carbon/human/H = O
+			if(H.nif && H.nif.flag_check(NIF_V_FLASHPROT,NIF_FLAGS_VISION))
+				H.nif.notify("High intensity light detected, and blocked!",TRUE)
+				continue
```
Comment (Novacat) [inline code/game/machinery/flasher.dm:73]: These changes need to be marked with //VOREStation Edit Start and //VOREStation Edit End, preferably above and below this block.

**Comment by Heroman3003** [comment]
It made sense in context that ALL reagents from the prey were transfered into bloodstream. I think they should actually ALL transfer to ingested instead.

**Comment by cadyn** [comment]
> It made sense in context that ALL reagents from the prey were transfered into bloodstream. I think they should actually ALL transfer to ingested instead.

Fair enough

**Pull request merged**